app/src/main/res/layout/nav_header.xml

```diff
@@ -13,17 +14,33 @@
     android:padding="15dp"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
-    <de.hdodenhof.circleimageview.CircleImageView
-        android:id="@+id/nav_header_imageView"
-        android:layout_width="55dp"
-        android:layout_height="55dp"
-        android:src="@drawable/icon_nav"
-        app:civ_border_color="@color/white"
-        app:civ_border_width="1dp"
-        android:layout_gravity="top|start"
-        android:layout_marginStart="5dp"
-        android:layout_marginTop="5dp"
-        />
+    <RelativeLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <de.hdodenhof.circleimageview.CircleImageView
+            android:id="@+id/nav_header_imageView"
+            android:layout_width="55dp"
+            android:layout_height="55dp"
+            tools:src="@drawable/icon_nav"
+            app:civ_border_color="@color/white"
+            app:civ_border_width="1dp"
+            android:layout_gravity="top|start"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="5dp"
+            />
+
+        <ImageView
```

Comment:
The spacing and alignment are not proper of imageView with circleImageView. Check that in Design editor. 
![oss](https://user-images.githubusercontent.com/34706326/59174103-9b9d8a80-8b6d-11e9-83cf-ab167a0a85d7.jpeg)
![sonu](https://user-images.githubusercontent.com/34706326/59174105-9cceb780-8b6d-11e9-9b50-a68019cafe06.jpeg)